### ISSUE TITLE
Pointnet regression

### ISSUE THREAD

**Issue by pieterbl86** [nikitakaraevv/pointnet#5]
@nikitakaraevv 

Thank you very much for this nice repository. I found it really insightful when learning about Pointnet. 

Suppose I want to do regression instead of classification. An example is to estimate the object's weight from a 3D point cloud. 

Can regression be achieved by "simply" changing the output activation layer to a linear one (instead of softmax) and implementing a different loss function (for example mean absolute error instead of negative log likelihood loss). 

I'm pretty new to Pointnet, so maybe you have some nice suggestions...
Thanks in advance!

**Comment by nikitakaraevv** [comment]
@pieterbl86 
Thank you for your feedback and for the question!
As PointNet learns both local and global structure of point clouds very well, it can solve other tasks related to point cloud recognition including, for instance, normal estimation which is a regression task.
So, I think that indeed you could simply change the activation layer and use a different loss function to solve a regression problem. It should work :)
